doc/tools/docgram/README.md

```diff
@@ -112,6 +112,8 @@ for documentation purposes:
 
 * `make doc_gram_rsts` updates the `*Grammar` and `.rst` files.
 
+* `DOCGRAMWARN=1 make doc_gram_rsts` will additionally print warnings.
```

Comment:
I find it strange to put the variable assignment at the beginning.  It seems to work, oddly enough, but I have no idea when the shell started to support that.

Reply:
I also tried at the end of the line, and oddly it didn't work for me. That's why I documented on the beginning of the line.

Reply:
Variable assignments done as make arguments may not be mutated without `override` https://www.gnu.org/software/make/manual/make.html#Override-Directive
`foo=bar cmd` is just an environment variable so does not need override.

IMO the logic at https://github.com/coq/coq/blob/b0e85d75f8085b7e94d976a8243b7efe48bf9d0d/Makefile.doc#L51-L56 should set a new variable instead of mutating.